classes/Smarty/SmartyResourceModule.php

```diff
@@ -31,6 +31,15 @@
  */
 class SmartyResourceModuleCore extends Smarty_Resource_Custom
 {
+    /**
+     * @var array<string>
+     */
+    public $paths;
+    /**
+     * @var bool
+     */
+    public $isAdmin;
```

Comment:
Without analyzing it further, it seems tricky to have `public $isAdmin.`

Reply:
You're right.  It seems not used and no really variables put in this one. So I remove it (creating a new BC Break (:arrow_up: documented)).